app/models/user.rb

```diff
@@ -2,7 +2,7 @@ class User < ActiveRecord::Base
   # Include default devise modules.
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable,
-         :confirmable, :omniauthable
```

Comment:
Do you want to have confirmations later? If you remove confirmable, does it stop generating confirmation tokens when users are created?